scintellometry/folding/fold.py

```diff
@@ -172,11 +172,10 @@ def fold(fh, comm, samplerate, fedge, fedge_at_top, nchan,
             freq = (freq_in[:, np.newaxis] + tb * u.Hz *
                     rfftfreq(oversample*2, dtsample.value/2.)[::2])
         # same as fine = rfftfreq(2*ntint, dtsample.value/2.)[::2]
-        fcoh = freq_in[np.newaxis, :] + tb * u.Hz * rfftfreq(
-            ntint*2, dtsample.value/2.)[::2, np.newaxis]
+        fcoh = freq_in - u.Hz * np.fft.fftfreq(ntint, dtsample.value)[:,np.newaxis]
```

Comment:
I think you still need `tb` in here, in that the frequencies will be different depending on what it is.